Lsunae/ElevenStreetSample/app/src/main/java/com/exam/elevenstreet/data/source/remote/ProductRemoteDataSourceImpl.kt

```diff
@@ -0,0 +1,49 @@
+package com.exam.elevenstreet.data.source.remote
+
+import android.os.AsyncTask
+import com.exam.elevenstreet.R
+import com.exam.elevenstreet.data.ProductCallback
+import com.exam.elevenstreet.data.parser.ProductXmlPullParserHandler
+import com.exam.elevenstreet.network.api.ElevenStreetApi
+import com.exam.elevenstreet.network.model.ProductResponse
+import com.exam.elevenstreet.view.product.App
+import com.exam.elevenstreet.view.product.ProductActivity
+
+import java.net.URL
+
+class ProductRemoteDataSourceImpl constructor(private var elevenStreetApi: ElevenStreetApi) : ProductRemoteDataSource{
```

Comment:
```suggestion
class ProductRemoteDataSourceImpl private constructor(private var elevenStreetApi: ElevenStreetApi) : ProductRemoteDataSource{
```
아래에 만든 getInstance함수로만 해당 클래스를 접근하게 하면 좋겠어요

Reply:
수정했습니다!
0b02636